langchain/src/vectorstores/supabase.ts

```diff
@@ -0,0 +1,184 @@
+/// Youre not allowed to run raw sql on supabase apis I dont beleive
```

Comment:
**Comment:** Briefly looked into this. Indeed, Supabase doesn't like SQL getting executed outside of the safety of their own environment.

This is very hacky, but maybe we can bypass supabase altogether and use the postgres CLI in the terminal?